controllers/services/auth/resources/admingroup-clusterrole.tmpl.yaml

```diff
@@ -18,6 +18,15 @@ rules:
   - auths/status
   verbs:
   - get
+- apiGroups:
+  - datasciencecluster.opendatahub.io
+  resources:
+  - datascienceclusters
+  verbs:
+  - get
+  - list
+  - watch
+  - update
```

Comment:
Do we also want to add `patch`? We plan to use this in the Dashboard to allow clusteradmins to edit some things in the DSC. Not sure if `update` is enough on it's own or if it's best to also add `patch`.

Reply:
Sure, makes sense. I've added patch.